src/renderer/components/dock/log-resource-selector.tsx

```diff
@@ -30,36 +33,61 @@ export const LogResourceSelector = observer((props: Props) => {
     reload();
   };
 
-  const getSelectOptions = (containers: IPodContainer[]) => {
-    return containers.map(container => {
+  const onPodChange = (option: SelectOption) => {
+    const selectedPod = podsStore.getByName(option.value, pod.getNs());
+
+    save({ selectedPod });
+  };
+
+  const getSelectOptions = (items: string[]) => {
+    return items.map(item => {
       return {
-        value: container.name,
-        label: container.name
+        value: item,
+        label: item
       };
     });
   };
```

Comment:
This seems like a piece of code that would be used in multiple places. Should it be moved to `renderer/utils`?

Reply:
Good idea, but maybe outside scope for this PR? I've seen it used inline even, wonder if it could be incorporated as default behaviour of `Select` component when adding options?

Reply:
Agreed to consider this issue in the separate PR.